src/network.cpp

```diff
@@ -281,7 +386,28 @@ namespace xiloader
                 std::cout << std::endl;
 
                 char event_code = (input == "1") ? 0x10 : 0x30;
-                sendBuffer[0x20] = event_code;
+
+                if (input == "3")
+                {
+                    std::string confirmed_password = "";
+                    do
+                    {
+                        std::cout << "Enter new password (6-15 characters): ";
```

Comment:
In various places we list 15/16/17 as the length of passwords, including/excluding null terminator, we should verify what goes where

Reply:
password is 32 characters now